### PR TITLE
Remove extraneous package, default role, focus style override

### PR DESCRIPTION
**Pull request by DrewWarrenTIY**

### CODE REVIEW

File: src/components/Modal.js [hunk 1]
```diff
@@ -218,7 +216,6 @@ class Modal extends React.Component {
                 onKeyUp={e => e.keyCode === 13 && this.props.onRequestClose()}
                 role='button'
                 style={styles.close}
-                tabIndex={0}
```
Comment (cerinman) [inline src/components/Modal.js]: While you're here.... Is the role='button' and onKeyUp really needed on this native button? Can we remove them too?

File: src/components/Modal.js [hunk 2]
```diff
@@ -204,7 +203,6 @@ class Modal extends React.Component {
               ref={ref => this._modalContent = ref}
               role={this.props.role}
               style={Object.assign({}, styles.content, this.props.contentStyle)}
-              tabIndex={0}
```
Comment (cerinman) [inline src/components/Modal.js]: Just to be safe. You didn't run into any console errors around assigning undefined to the role here on line 204 above now that the default has been removed?

Comment (DrewWarrenTIY) [reply src/components/Modal.js]: Double checked with passing no role prop. No console errors

File: src/components/Modal.js (hunk 1 above)
Comment (DrewWarrenTIY) [reply src/components/Modal.js]: Removed

File: src/components/Modal.js [hunk 3]
```diff
@@ -204,7 +203,7 @@ class Modal extends React.Component {
...
+              tabIndex={-1} // DO NOT REMOVE: Focus will default to wrapper with focusOnLoad prop
```
Comment (cerinman) [inline src/components/Modal.js]: DO NOT REMOVE: ?

**Comment by cerinman** [comment]
I don't see a :shipit:. Why did you merge?

**Comment by cerinman** [comment]
Also, @cgregs32 will need to add his MR to the release you will do with this. https://github.com/mxenabled/mx-react-components/pull/830 Make sure you two work together on the changelog update.

File: src/components/Modal.js (hunk 3 above)
Comment (cerinman) [reply src/components/Modal.js]: Never mind on this one. I refreshed and realized you added it.

**Pull request merged**